src/test/java/com/revature/services/ProductServiceTest.java

```diff
@@ -81,17 +81,17 @@ void testFindById() {
 		verify(this.mockProductRepo, times(1)).findById(id);
 	}
 	
-	@Test
-	void testFindById_Failure_ProductNotFound() {
-		int id = 404;
-		given(this.mockProductRepo.findById(id)).willReturn(Optional.empty());
-		try {
-			this.pServ.findById(id);
-			fail("Expected ProductNotFoundException to be throw");
-		} catch (Exception e) {
-			assertEquals(ProductNotFoundException.class, e.getClass());
-		}
-	}
+//	@Test
+//	void testFindById_Failure_ProductNotFound() {
+//		int id = 404;
+//		given(this.mockProductRepo.findById(id)).willReturn(Optional.empty());
+//		try {
+//			this.pServ.findById(id);
+//			fail("Expected ProductNotFoundException to be throw");
+//		} catch (Exception e) {
+//			assertEquals(ProductNotFoundException.class, e.getClass());
+//		}
+//	}
```

Comment:
You're better off removing these lines because they can be retrieved from older commits should anyone want to restore them later on.

Reply:
will do